client/app/common/navbar/navbar.html

```diff
@@ -11,13 +11,13 @@
             </li>
           </ul>
         </div>
-        <ul class="nav navbar-nav" ng-if="$ctrl.auth.getAuthState()">
+        <!-- <ul class="nav navbar-nav" ng-if="$ctrl.auth.getAuthState() && UsersCrtl.subTitle">
```

Comment:
Did the subTitle check not work in the controller? We should either remove this if we're not going to use it, or structure the if in such a way that lets us enable it when modules call for it.